### PR TITLE
cmd/listplans: Make tabular default, added test for tabular output

### PR DESCRIPTION
**Pull request by mattyw**

### CODE REVIEW

File: cmd/listplans/list_plans_test.go [hunk 1]
```diff
@@ -55,6 +55,64 @@ func (s *ListPlansCommandSuite) SetUpTest(c *gc.C) {
...
+			ResolvedURL: "series/some-charm-url",
+			Stub:        s.stub,
+		},
+	}
+	defer listPlans.Close()
+	ctx, err := cmdtesting.RunCommand(c, listPlans, "some-charm-url")
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(cmdtesting.Stdout(ctx), gc.Equals,
+		`PLAN             	PRICE	DESCRIPTION                                       
+bob/test-plan-1  	     	Lorem ipsum dolor sit amet, consectetur adipiscing
```
Comment (cmars) [inline cmd/listplans/list_plans_test.go]: I think you'll see the text wrapping issue if you use a plan desc that has newline breaks in it (use yaml `|` instead if `>-`).

**Comment by cmars** [comment]
LGTM!

File: cmd/listplans/list_plans.go [hunk 2]
```diff
@@ -189,7 +189,8 @@ func formatTabular(value interface{}) ([]byte, error) {
 
 	table.AddRow("PLAN", "PRICE", "DESCRIPTION")
 	for _, plan := range plans {
-		table.AddRow(plan.URL, plan.Price, plan.Description)
+		description := strings.Replace(plan.Description, "\n", " ", -1)
```
Comment (tasdomas) [inline cmd/listplans/list_plans.go]: Didn't we discuss leaving multiple (>=2) linebreaks in place?

**Comment by mattyw** [comment]
I think we need to discuss this in the standup

**Comment by cmars** [comment]
LGTM

**Comment by mattyw** [comment]
$$merge$$

**Pull request merged**